### PR TITLE
Integrate public profile with assignment history and current assignment

### PR DESCRIPTION
**Pull request by mjoyce91**
~~Need to fix tests~~

Use with https://github.com/MetaPhase-Consulting/State-TalentMAP-API/pull/313

### CODE REVIEW

File: src/Components/ProfileDashboard/PositionInformation/PositionInformation.jsx [hunk 1]
```diff
@@ -9,11 +9,12 @@ import StartEnd from './StartEnd';
...
   let assignmentEndDate;
-  assignmentEndDate = (assignment.status === 'active') ? assignment.estimated_end_date : assignment.end_date;
+  assignmentEndDate = (assignment.status === 'active' && assignment.estimated_end_date) ?
+    assignment.estimated_end_date : assignment.end_date;
   assignmentEndDate = assignmentEndDate ? formatDate(assignmentEndDate) : '';
 
   // format our values
-  const formattedAssignmentDates = assignment.start_date && assignment.estimated_end_date ?
+  const formattedAssignmentDates = assignmentStartDate && assignmentEndDate ?
     (<StartEnd
```
Comment (elizabeth-jimenez) [inline src/Components/ProfileDashboard/PositionInformation/PositionInformation.jsx]: ▶️ i like the arrow

File: src/Components/ProfileDashboard/PositionInformation/PositionInformation.jsx [hunk 2]
```diff
@@ -9,11 +9,12 @@ import StartEnd from './StartEnd';
...
       start={assignmentStartDate}
       end={assignmentEndDate}
```
Comment (elizabeth-jimenez) [inline src/Components/ProfileDashboard/PositionInformation/PositionInformation.jsx]: can do lodash get on lines 23, 24, 25, but not a must.

**Comment by elizabeth-jimenez** [comment]
just have to resolve the conflicts 🥇

File: src/Components/ProfileDashboard/PositionInformation/PositionInformation.jsx (hunk 2 above)
Comment (mjoyce91) [reply src/Components/ProfileDashboard/PositionInformation/PositionInformation.jsx]: yeah, that's an old way I used to do it. I'll leave it for now

**Pull request merged**